.github/workflows/chart-streams.yaml

```diff
@@ -14,20 +14,25 @@ jobs:
       - uses: actions/cache@v2
         id: cache-libgit2
         with:
-          path: /usr/local/lib/libgit2.so.*
-          key: ${{ runner.os }}-libgit2-1.2.0
-      
+          path: /opt/libgit2
+          key: ${{ runner.os }}-${{ hashFiles('hack/ubuntu/libgit2.sh') }}
+
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
           go-version: '1.16.8'
 
       - name: "Install libgit2 from sources"
-        run: ./hack/ubuntu/libgit2.sh
+        run: sudo -E ./hack/ubuntu/libgit2.sh
```

Comment:
I wonder if it should become a `Makefile` target, thus we take central control of environment variables that determines the version of things, like in this case `libgit2`.